controller/src/beerocks/cli/beerocks_cli_ucc_listener.cpp

```diff
@@ -0,0 +1,169 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "beerocks_cli_ucc_listener.h"
+#include "beerocks_cli_bml.h"
+
+#include <beerocks/bcl/beerocks_string_utils.h>
+
+#define SELECT_TIMEOUT_MSC 100
+#define NLF std::endl << std::flush // New Line Flush
```

Comment:
For debug reasons. Since it is a multithreaded file, I wanted to make sure that the prints will be flushed.

I looked here and it doesn't say explicitly that std::endl flushes the output.
https://en.cppreference.com/w/cpp/io/manip/endl

Reply:
Turns out I was wrong - it is std::cerr that is flushed automatically, and not just on endl but on any operation: https://en.cppreference.com/w/cpp/io/manip/unitbuf